### PR TITLE
gitmate_gitlab.md: Fix typo and broken links

### PR DESCRIPTION
**Pull request by Kaushl2208**

### CODE REVIEW

File: _projects/gitmate_gitlab.md [hunk 1]
```diff
@@ -3,8 +3,7 @@ collaborating_projects:
   - coala
   - GitMate
   - IGitt
```
Comment (srivama) [inline _projects/gitmate_gitlab.md]: Please fix the typo here. There isn’t a need to remove the entire line.

File: _projects/gitmate_gitlab.md [hunk 2]
```diff
@@ -3,8 +3,7 @@ collaborating_projects:
...
-desc: "GitMate will support GitHub very soon with it's plugin system - GitLab and Bitbucket are not far away thanks to
-       your help!"
+desc: "GitMate will support GitHub very soon with it's plugin system - GitLab and Bitbucket are not far away"
```
Comment (srivama) [inline _projects/gitmate_gitlab.md]: Please fix the type here instead of removing the entire line :)

**Comment by jayvdb** [comment]
Please use `git` command line to solve the conflict and rebase.  Please do not use any GitHub buttons, or CLI merge operations -- see https://coala.io/git and https://coala.io/newcomer for more info

**Comment by jayvdb** [comment]
On the last line, `Fixes` -> `Closes` as this isnt a 'bug' .  The URL was correct, but the repo was moved.

Most important, there must be a blank line before the `Closes ...` line

**Comment by jayvdb** [comment]
Hi, you have done a merge, not a squash and rebase.

You can find some tips on how to fix this at https://coala.io/git , or ask for help at https://coala.io/chat .

**Pull request closed without merging**